torch/_torch_docs.py

```diff
@@ -1857,8 +1857,10 @@ def merge_dicts(*dicts):
 Creates a :class:`Tensor` from a :class:`numpy.ndarray`.
 
 The returned tensor and :attr:`ndarray` share the same memory. Modifications to
-the tensor will be reflected in the :attr:`ndarray` and vice versa. The returned
-tensor is not resizable.
+the tensor will be reflected in the :attr:`ndarray` and vice versa. The
+returned tensor is not resizable. The :attr:`ndarray` must have a dtype of
```

Comment:
Please create a newline when you start introducing the dtypes.

Also, I don't think we need to (or should) specify the type of Error returned; I don't think we really need to commit to that nor do I think most users will care that much.

Also, I would use the wording:
"currently supported dtypes are .." because this is the sort of thing that will certainly become out of date (as you discovered!), and using less strong wording is preferable there.

Reply:
Thanks, I have pushed the fix.